docs/topic-guides/blendable-web-apps/view-model-attaching.md

```diff
@@ -108,5 +108,5 @@ Here is the explanation of each field:
 * `Token`: Attachment rule token described above. Default value is an empty string which is treated as "DefaultToken".
 * `Active`: Defines default attachment rule rule activeness. Default value is "True".
 * `Contexts`: Attachment rule contexts (if any). Default value is "null".
-* `AllowFromDirection`: Allows attachment rule calls from this URI (it defines if this URI can trigger attachment process). Default value is "True".
-* `AllowToDirection`: Allows attachment rule calls to this URI (it defines if this URI can be triggered during attachment process). Default value is "True".
+* `AllowFromDirection`: Allows attachment rule calls from this URI (it defines if this URI can trigger attachment process). That means if app calls `Self.GET()` on attachment rule URI it will/will not request other apps for content with the same contexts. Default value is "True".
```

Comment:
"app" => "an app"

"attachment rule URI" => "an attachment rule URI"

I'm not sure what you mean with the "will/ill not" construct. Can you clarify?

Reply:
I wanted to point out true/false situations. Could you propose better way to describe it? I wanted to just point that when attribute is true it will, and if its false it will not.

Reply:
Try something like this:

> If it's set to "true" it..., otherwise it..

Reply:
Take a look now, I changed it a bit.